client/src/components/reader/ReturnMenuButton.js

```diff
@@ -0,0 +1,29 @@
+import React, { Component, PropTypes } from 'react';
+
+export default class AppearanceMenuButton extends Component {
+
+  static propTypes = {
+    toggleReaderMenu: PropTypes.func,
+    active: PropTypes.bool
```

Comment:
Why does this prop exist if it doesn't do anything? See above, where it's passed in as well.